coalib/bearlib/__init__.py

```diff
@@ -4,15 +4,25 @@
 while offering the best possible flexibility.
 """
```

Comment:
This file contains unused source code.

*Origin: PyUnusedCodeBear, Section: `flakes`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp7ip5_d4o/coalib/bearlib/__init__.py
+++ b/tmp/tmp7ip5_d4o/coalib/bearlib/__init__.py
@@ -4,7 +4,6 @@
 while offering the best possible flexibility.
 """
 
-import cProfile
 import logging
 
 from coalib.settings.FunctionMetadata import FunctionMetadata
```